ext/spl/spl_iterators.stub.php

```diff
@@ -403,10 +453,16 @@ public function __construct(
         int $mode = self::SELF_FIRST
     ) {}
 
-    /** @return void */
+    /**
+     * @return void
+     * @alias RecursiveIteratorIterator::rewind
+     */
     public function rewind() {}
```

Comment:
Could we just inherit these instead?

Reply:
Sure! I was also wondering about it for a while, but I was too busy copy-pasting code so that haven't actually checked it yet.